docs/build.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
please use oracle copyright

Reply:
You cannot change the copyright of existing work. The copyright must remain intact.

Reply:
I'll add a comment in the file "build.sh copied unmodified from xxx repository"

Reply:
Note that this isn't required, so I've approved this PR as it's fine as-is.